lib/Sema/CSApply.cpp

```diff
@@ -6099,12 +6099,17 @@ static bool hasCurriedSelf(ConstraintSystem &cs, ConcreteDeclRef callee,
 /// Apply the contextually Sendable flag to the given expression,
 static void applyContextualClosureFlags(Expr *expr, bool implicitSelfCapture,
                                         bool inheritActorContext,
```

Comment:
This function now takes 6 boolean arguments. Please turn it into an OptionSet

Reply:
@xedin pulled this into his https://github.com/swiftlang/swift/pull/81496/

Seems he changed it to be a `ParameterListInfo` there already 👍